docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -153,6 +153,13 @@ There you can find links to upgrade notes for other versions too.
     - and include `generate-build-version` and `clean-redis-old` to your build phing targets. Please find inspiration in [#886](https://github.com/shopsys/shopsys/pull/886/files)
     - once you finish this change (include the `build-version` into caches), you still should deal with older redis cache keys that don't use `build-version` prefix (16 digits).
       Such keys are not removed even by `clean-redis-old`, please find and remove them manually (via console or UI)
+- skip `Shopsys\CodingStandards\Sniffs\ForceLateStaticBindingForProtectedConstantsSniff` sniff in your `easy-coding-standard.yml`, so you won't be forced to use `static` on protected constants ([#904](https://github.com/shopsys/shopsys/pull/904))
+    - ```yaml
+        parameters:
+            skip:
+                Shopsys\CodingStandards\Sniffs\ForceLateStaticBindingForProtectedConstantsSniff:
+                    - '*'
+      ```
```

Comment:
Do we want to make the rule default in our coding-standards? Maybe we can add it only to our extensible packages (or better yet, make an `extensible_easy_coding_standards.yml` ruleset for extensible packages) so it requires no change from the project developers.

Reply:
That would be nice, but I didn't find a way how to achieve this from a view of a monorepo. 
When I run ecs with a monorepo configuration, it uses all the configurations and if I don't skip the sniff in a project base, I end up with different rules when installing monorepo and project-base itself.

Reply:
Yeah, that makes sense. It would mean we have to do separate runs of the checking in monorepo or configure monorepo to skip it in the whole project-base directory (or namespace).

@vitek-rostislav: This might be a reason why the `ForbiddenDoctrineInheritanceSniff` was configured the way it was as you've mentioned the other day on a team meeting.

Reply:
I think that all sniffs/fixers created during this PR should be included in standard configuration.
The coding-standards package is
> Tools for automated checks of coding standards in PHP projects. Predefined rulesets included.

The rulesset is predefined for the Shopsys usage. I think all these sniffs/fixes fits for the SSFW and also for the Shopsys agency. And if there is a user that doesn't want to use such sniff, he can freely skip the sniff/fixer.

Reply:
Please move the instruction to section `coding-standards`. Once you move it, you don't have to use the example with `yaml` configuration

Reply:
I'm not really sure if I want to force this specific fixer.
For me, it lies in a very blurry area. We want this fixer in the framework to allow users who extend the classes to change the behavior to match their needs. 
It helps with our specific use-case. 

But in a project, I don't think it's a necessary rule and I don't believe it should be forced. 
And I don't see any added value of forcing this rule in a project, do you?

Reply:
What about moving instructions to `coding-standards` section?

Reply:
I said my point of view. If you will agree with someone else that a sniff won't be part of our default coding-standards, it's ok for me... Please let me know ;-)